.vscode/settings.json

```diff
@@ -1,5 +1,6 @@
 {
   "editor.formatOnSave": true,
   "prettier.singleQuote": true,
+  "prettier.trailingComma": "none",
```

Comment:
I think this is just making explicit what was previously implicit.

I added this to the workspace settings when I noticed that my user settings were changing the formatting in this project.